hadoop-hdds/docs/content/feature/Observability.zh.md

```diff
@@ -0,0 +1,222 @@
+---
+title: "可观察性"
+weight: 8
+menu:
+   main:
+      parent: 特性
+summary: 用于增强 Ozone 可观察性的不同工具
+---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+Ozone 为获取集群运行状态的详细信息提供了多种工具。
+
+## Prometheus
+
+Ozone 原生支持 Prometheus 集成。所有内部指标（通过 Hadoop 指标框架收集的指标）都发布在 `/prom` HTTP 端点下（比如 SCM 的指标发布在 http://localhost:9876/prom 下）。
+
+Prometheus 端点默认开启，但可以通过 `hdds.prometheus.endpoint.enabled` 配置参数关闭。
+
+在页面开启了 SPNEGO 认证的安全环境中，Prometheus 无法直接访问，需要配置特定的认证 token 才可以进行监控。
+
+ `ozone-site.xml` 示例：	
+
+```XML
+<property>
+   <name>hdds.prometheus.endpoint.token</name>
+   <value>putyourtokenhere</value>
+</property>
+```
+
+Prometheus 配置示例:
+
+```YAML
+scrape_configs:
+  - job_name: ozone
+    bearer_token: <putyourtokenhere>
+    metrics_path: /prom
+    static_configs:
+     - targets:
+         - "127.0.0.1:9876" 
+```
+
+## 分布式追踪
+
+分布式追踪可以通过可视化端到端性能来辅助定位性能瓶颈。
+
+Ozone 使用 [jaeger](https://jaegertracing.io) 追踪库来收集追踪数据并发送到任何兼容的后端（Zipkin等）
+
+追踪默认是关闭的，但可以通过 `ozone-site.xml` 中的 `hdds.tracing.enabled` 参数开启：
+
+```XML
+<property>
+   <name>hdds.tracing.enabled</name>
+   <value>true</value>
+</property>
+```
+
+根据[此处文档](https://github.com/jaegertracing/jaeger-client-java/blob/master/jaeger-core/README.md)，Jaeger 客户端可以通过环境变量进行配置，例如：
+
+```shell
+JAEGER_SAMPLER_PARAM=0.01
+JAEGER_SAMPLER_TYPE=probabilistic
+JAEGER_AGENT_HOST=jaeger
+```
+
+此配置会记录 1% 的请求来限制性能开销。更多有关 jaeger 采样的信息[请查阅文档](https://www.jaegertracing.io/docs/1.18/sampling/#client-sampling-configuration)
```

Comment:
The English sentence is `This configuration will record 1% of the requests to limit the performance overhead`. It is easy to translate but not easy to understand for me. @elek Could you explain a little bit about `limit the performance overhead`?